.github/workflows/windows-msys2.yml

```diff
@@ -249,22 +224,6 @@ jobs:
         run: |
             pacman --noconfirm -U https://repo.msys2.org/mingw/${{matrix.conf.sys}}/mingw-w64-clang-${{matrix.conf.arch}}-SDL2-2.26.5-1-any.pkg.tar.zst
 
-      - name:  Prepare compiler cache
-        id:    prep-ccache
-        run: |
-          mkdir -p "${CCACHE_DIR}"
-          echo "dir=$(cygpath -w $CCACHE_DIR)"         >> $GITHUB_OUTPUT
-          echo "today=$(date -I)"                      >> $GITHUB_OUTPUT
-          echo "yesterday=$(date --date=yesterday -I)" >> $GITHUB_OUTPUT
-
-      - uses:  actions/cache@v4.0.1
-        id:    cache-ccache
-        with:
-          path: ${{ steps.prep-ccache.outputs.dir }}
-          key:  ccache-msys2-${{matrix.conf.sys}}-release-${{ steps.prep-ccache.outputs.today }}-3
-          restore-keys: |
-            ccache-msys2-${{matrix.conf.sys}}-release-${{ steps.prep-ccache.outputs.yesterday }}-3
-
```

Comment:
Just commenting because you've touched the MSYS2 workflow. You might want to add the new version handling to this as well:

```
      - name: Inject version string
        run: |
          set -x
          VERSION=$(./scripts/get-version.sh version-and-hash)
          echo "VERSION=$VERSION" >> $GITHUB_ENV

```